src/nmo.js

```diff
@@ -49,6 +49,8 @@ nmo.load = function load (opts) {
       .load(opts)
       .then((config) => {
         nmo.config = config;
+        nmo.config.set('name', 'nmo');
+        nmo.config.set('help-directory', `${__dirname}/..`);
```

Comment:
we might want include a separate template which is then loaded here https://github.com/garrensmith/couchdb-nmo/blob/small-fixes/bin/nmo-cli.js#L24